### PR TITLE
User/graeme ford

### PR DESCRIPTION
**Pull request by G-kodes**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# SOURCE AND CREDITS: https://github.com/chrishah/samtools-docker/blob/master/Dockerfile
+
+FROM alpine:latest
+LABEL version="v1.0" description="A docekr image for the Pharmacogenetics ANalysis Pipeline" maintainer="Graeme Ford <Graeme.Ford@tuks.co.za>"
```
Comment (MeganHolborn) [inline Dockerfile:4]: Change spelling of "docker" and "analysis"

File: workflow/Snakefile [hunk 2]
```diff
@@ -193,22 +194,22 @@ rule FILTER:
         queue=search("queue", "FILTER"),
         walltime=search("walltime", "FILTER"),
     run:
-        # Filter out variants with >= 20% missingness
+        # Filter out variants with >= 100% missingness
         shell(
-            "module load plink-2; plink2 --allow-extra-chr --vcf {input} --geno 0.2 --output-chr chr26 --export vcf-4.2 bgz --out results/FILTER/ALL_FILTERED1"
+            "module load plink-2; plink2 --allow-extra-chr --vcf {input} --geno 1 --output-chr chr26 --export vcf-4.2 bgz --out results/FILTER/ALL_FILTERED1"
```
Comment (MeganHolborn) [inline workflow/Snakefile:199]: Is a decimal point necessary after the "1"?

File: workflow/Snakefile [hunk 3]
```diff
@@ -193,22 +194,22 @@ rule FILTER:
...
         ),
         # Filter out variants where the reference in the file does not match that of the reference genome
         shell(
             "module load bcftools-1.7; bcftools norm --check-ref x results/FILTER/ALL_FILTERED1.vcf.gz --fasta-ref {params.ref} -o results/FILTER/ALL_FILTERED2.vcf.gz -O z"
         ),
-        # Filter out samples with >= 20% missingness
+        # Filter out samples with >= 100% missingness
         shell(
-            "module load plink-2; plink2 --allow-extra-chr --vcf results/FILTER/ALL_FILTERED2.vcf.gz --mind 0.2 --output-chr chr26 --export vcf-4.2 bgz --out results/FILTER/ALL_FILTERED3"
+            "module load plink-2; plink2 --allow-extra-chr --vcf results/FILTER/ALL_FILTERED2.vcf.gz --mind 1 --output-chr chr26 --export vcf-4.2 bgz --out results/FILTER/ALL_FILTERED3"
```
Comment (MeganHolborn) [inline workflow/Snakefile:207]: Is a decimal point necessary after the "1"?

**Review by MeganHolborn — APPROVED** [review]

File: workflow/Snakefile (hunk 2 above)
Comment (G-kodes) [reply workflow/Snakefile:199]: I have downloaded Plink-2.0 on my local Windows and tested. No decimal is required for 100% missingness tests thankfully.

File: workflow/Snakefile (hunk 3 above)
Comment (G-kodes) [reply workflow/Snakefile:207]: I have downloaded Plink-2.0 on my local Windows and tested. No decimal is required for 100% missingness tests thankfully.

File: Dockerfile (hunk 1 above)
Comment (G-kodes) [reply Dockerfile:4]: The spelling corrections have been made. Thank you very much for catching that!

**Pull request merged**